start-training

```diff
@@ -133,24 +137,33 @@ try:
         if args.wandb:
             import wandb
             if wandb.login():
+                entity = "safelife"
```

Comment:
I was originally thinking that we'd rely on `wandb/settings` to set all of this info. Is there a reason to set it within the `start-training` script instead? I want to make sure that we're not adding code that's overly specific to our workflow.

In any case, we shouldn't have `entity` hard-coded here because only you and I have access to the wandb safelife entity.

Reply:
FWIW, my `wandb/settings` file looks like this:
```
[default]
entity = safelife
project = clw-hacking
base_url = https://api.wandb.ai
```
and that seems to get everything in the right spot.

Reply:
Oh, I think I didn't absorb the nature of that file properly during the call. I somewhat like the property of runs being naturally assorted to projects based on `env_type`, without needing a file edit to launch them correctly, but aside from that I don't have opinions on whether/how to do this!

Reply:
Ok, let's revert this for the time-being then and just specify things via the settings file. Seems like wandb should have smart filters based on hyperparameters anyways.